pom.xml

```diff
@@ -100,11 +102,10 @@
             <artifactId>metrics-jvm</artifactId>
             <version>${io.dropwizard.metrics5.version}</version>
         </dependency>
-
         <dependency>
             <groupId>com.wavefront</groupId>
             <artifactId>wavefront-sdk-java</artifactId>
-            <version>0.9.0</version>
+            <version>0.9.1-SNAPSHOT</version>
```

Comment:
Add a TODO here that we will switch from SNAPSHOT to a released dependency

Reply:
done.